test_driver/app_test.dart

```diff
@@ -25,32 +29,44 @@ void main() {
     await driver.waitFor(find.text('Finance'));
     await driver.waitFor(find.text('Goals'));
 
-    await screenshot(driver, config, 'home_screen');
+    await takeScreenshot(driver, 'home/home_screen');
   }, timeout: Timeout(Duration(seconds: 60)));
 
-  test('can navigate to each base screen', () async {
+  test('adding a task', () async {
     await driver.tap(find.text('Tasks'));
-    await screenshot(driver, config, 'tasks_screen');
-    await driver.tap(find.byType('BackButton'));
+    await takeScreenshot(driver, 'tasks/tasks_screen');
+
+    await driver.tap(find.byType('FloatingActionButton'));
+    await takeScreenshot(driver, 'tasks/add_task_button_pressed');
 
-    await driver.tap(find.text('Calendar'));
-    await screenshot(driver, config, 'calendar_screen');
-    await driver.tap(find.byType('BackButton'));
+    await driver.waitUntilNoTransientCallbacks();
```

Comment:
Still getting the 'keyboard not showing' error sometimes during tests.  Need to investigate a sure-fire way to get the keyboard to show